api/books/booklist.ts

```diff
@@ -13,7 +13,6 @@ export const getBooks = async ({ auth }: GetBooksProps) => {
   })
     .then((res) => {
       const books = res.data
-      localStorage.setItem("books", JSON.stringify(books))
       return books
     })
     .catch((error) => {
```

Comment:
@normancapule wishes to add error handling for this, just following up.

Reply:
yep. still not used to handling errors on the front-end. will need to research about it first.

Reply:
Like my comment about shared headers, feel free to create a shared error handling function